app/controllers/admins/dashboard_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Admins::DashboardController < ApplicationController
+  def index
```

Comment:
The admin dashboard is the page where the admin can search and sort through all of the companies that have signed up. They will be able to mark a company as contacted and so on...